Bga 316 change hard slice for keys

silalvado opened the pull request:
Add a method to remove prefix in private and public keys.

silalvado commented:
> This looks good overall - do we need to change the pub in ACCOUNT_1 would be my only question?

We have two accounts, one with prefix and one without prefix. We need both to test different things.

twtaylorbitgo commented:
That sounds good to me - can you add a comment along those lines to each account or their tests? Just so its explicit

silalvado commented:
> That sounds good to me - can you add a comment along those lines to each account or their tests? Just so its explicit

I think you're right. I added comments in resources file and I modified const names to identify the keys with prefix in the unit tests.

twtaylorbitgo commented on modules/account-lib/test/unit/coin/hbar/transaction.ts:
```diff
@@ -7,6 +7,9 @@ import { KeyPair } from '../../../../src/coin/hbar/keyPair';
 describe('Hbar Transaction', () => {
   const coin = coins.get('thbar');
 
+  /**
+   *
+   */
```
NIT: Let's remove this in the next PR @gonzaalvarez70 @silalvado
